kms/pkcs11/pkcs11_test.go

```diff
@@ -41,6 +41,18 @@ func TestNew(t *testing.T) {
 		return k.p11, nil
 	}
 
+	var (
+		wantMissingModule    *PKCS11
+		wantErrMissingModule = true
+	)
+	if findP11KitProxy(context.Background()) != "" {
+		wantMissingModule = k
+		wantErrMissingModule = false
+	}
```

Comment:
This would be a more accurate test if the output of the shell command were mocked instead.

Reply:
I'll use `var findP11KitProxy = func(...) {...}`

Reply:
Fixed with 237a370